packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -3,16 +3,55 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import 'terra-base/lib/baseStyles';
 import Input from 'terra-form/lib/Input';
+import ButtonGroup from 'terra-button-group';
+
 import TimeUtil from './TimeUtil';
 import styles from './TimeInput.scss';
 
 const cx = classNames.bind(styles);
 
+const validDateRegex = new RegExp('^(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$');
+
+const FORMAT_12_HOUR = '12';
+const FORMAT_24_HOUR = '24';
+
+const inputType = {
+  HOUR: 0,
+  MINUTE: 1,
+};
+
+const keyCodes = {
+  BACKSPACE: 8,
+  ARROWLEFT: 37,
+  ARROWUP: 38,
+  ARROWRIGHT: 39,
+  ARROWDOWN: 40,
+  DELETE: 46,
+};
+
+const isConsideredMobileDevice = () =>
```

Comment:
I feel like this should be moved to the TimeUtils file.

Reply:
Moved. https://github.com/cerner/terra-core/pull/1027/commits/73c3583547a45c29ebd01fde64887c93f443c486